ui/src/components/HookForm/index.jsx

```diff
@@ -594,6 +647,76 @@ export default class HookForm extends Component {
               </ListItem>
             </Fragment>
           )}
+          <div className={classes.inputWrapper}>
+            <List
+              className={classes.inputList}
+              subheader={
+                <ListSubheader className={classes.subheader}>
+                  Bindings
+                </ListSubheader>
+              }>
+              <ListItem>
+                <ListItemText
+                  primary={
+                    <TextField
+                      required
+                      label="Pulse Exchange"
+                      name="pulseExchange"
+                      placeholder="exchange/<username>/some-exchange-name"
+                      onChange={this.handleInputChange}
+                      fullWidth
+                      value={pulseExchange}
+                    />
+                  }
+                />
+              </ListItem>
+              <ListItem>
+                <ListItemText
+                  primary={
+                    <TextField
+                      required
+                      label="Routing Key Pattern"
+                      placeholder="#.some-interesting-key.#"
+                      name="pattern"
+                      onChange={this.handleInputChange}
+                      fullWidth
+                      value={pattern}
+                    />
+                  }
+                />
+              </ListItem>
```

Comment:
There seems to be a lot of duplicated logic with `localhost:5080/pulse-messages`. I was thinking maybe we could create a `PulseBindings` component that will take care of creating

![Screen Shot 2019-05-13 at 10 12 36 AM](https://user-images.githubusercontent.com/3766511/57628456-fc827480-7567-11e9-9051-b86812571821.png)

`component/HookForm` and `views/PulseMessages` would then use that component and feed the appropriate props. Maybe something like:

```jsx
<PulseBindings bindings={...} onBindingAdd={this.handleBindingAdd} onBindingRemove={this.handleBindingRemove} ... />
```

What do you think?

Reply:
Actually, I was thinking the same, I was thinking of using existing `PulseMessages` but that does not seem to help.
I didn't think of creating a new component.
This sounds good. Let me work on it.

Reply:
Awesome, thanks @arshadkazmi42.